test/IO/test_opdx.py

```diff
@@ -133,7 +133,7 @@ def test_read_header(self):
         self.assertEqual(channel_1.info['Height_value'], 35.85522403809594)
         self.assertEqual(channel_1.info['z_scale'], 78.592625)
 
-        assert channel_1.info['unit'] == 'µm'  # see GH 281
```

Comment:
Shouldn't we leave this in the test, until we really remove `unit` from `info`? 

Which GH 281 is meant here - SurfaceTopography is around 100?

Reply:
I guess this is from the old PyCo issue tracker: https://github.com/pastewka/PyCo/issues/100 - I've readded the test